pom.xml

```diff
@@ -40,6 +40,12 @@
 				</exclusion>
 			</exclusions>
 		</dependency>
+		<dependency>
+			<groupId>com.vaadin.external.google</groupId>
+			<artifactId>android-json</artifactId>
```

Comment:
I've never seen this package before; is it specific to android development?  The first json package I thought of was the https://mvnrepository.com/artifact/org.json/json/20200518 package

Reply:
Yeah I guess that should be the correct one, I should be more careful when importing :)

Reply:
Changed :+1: